vmm/src/vm.rs

```diff
@@ -490,6 +492,29 @@ impl UserIoapicIrq {
         UserIoapicIrq { ioapic, irq }
     }
 }
+struct WinSize {
```

Comment:
You can get rid of this top-level struct.

Reply:
Done.